### DISCUSSION TITLE
Is having a Query Node in the roadmap ?

### DISCUSSION THREAD

**Discussion by EricDallAgnol** [atoti/atoti#698] [Q&A] [answered]
Hi 👋 

Using on my project several cubes, I would like to know if it is planned to introduce the Query Node in Atoti - see [here](https://www.activeviam.com/activepivot/latest/docs/distributed/distributed_intro/#query-cubes)
Such feature is really helpful especially for some reporting/export activities. I found it very convenient in the FRTB accelerator !

I know this [question has already been asked several months ago](https://github.com/atoti/atoti/discussions/496), but given that Atoti is becoming more "mature", I open this discussion again ^^ 

Many thanks,

Eric

**Comment by tibdex** [comment]
Hi Eric, there is a [private API to handle distribution](https://docs.atoti.io/0.7.0/releases/0.6.3.html#removed). It's limited to horizontal distribution though: polymorphic distribution is not supported. What kind of distribution do you have in mind?

**Comment by EricDallAgnol** [comment, marked as answer]
Hi Thibault,

Thanks for your reply.
I have in mind a polymorphic distribution given that we are creating cubes per application : VaR cube, Sensitivity Cube etc... and it would be great to have a consolidated view for some of these applications

**Reply by tibdex** [reply]
It's not a definitive decision but polymorphic distribution using the Python API is not on the roadmap.

**Reply by EricDallAgnol** [reply]
Ok thanks for your answer Thibault !
